met/src/basic/vx_log/string_array.h

```diff
@@ -29,7 +29,7 @@
 class StringArray {
 
    public:
-
+   
```

Comment:
```suggestion
   
```

Reply:
It won't let me commit this change. Says suggestion invalid because no changes were made to code.